invenio_search_ui/assets/semantic-ui/js/invenio_search_ui/components/SearchApp.js

```diff
@@ -68,7 +68,6 @@ export const SearchApp = ({ config, appName }) => {
         >
           <Overridable id={"SearchApp.layout"}>
             <Container>
-              <Grid relaxed padded>
```

Comment:
Minor: Is the ``<Grid>`` not required to have a ``<Grid.row>`` element below?

Reply:
From what I can see the page stays (besides the white space removal) with or without `<Grid>`.
With:
<img width="1680" alt="with" src="https://user-images.githubusercontent.com/6940589/97848017-c6532a80-1cf0-11eb-87c7-b0eced558eaa.png">


Without:
<img width="1680" alt="without" src="https://user-images.githubusercontent.com/6940589/97848043-d0752900-1cf0-11eb-98d5-cc0900a14ed4.png">

Reply:
@cenouralm to be "semantically" correct we should use row and columns even if it works otherwise...See [here](https://react.semantic-ui.com/collections/grid/#types-divided-number) an example! My point is, that following the correct structure that semantic-ui instructs, it would be easier to customize it afterwards :)